00_hello_world/main.go

```diff
@@ -8,6 +8,21 @@ import (
 
 var out io.Writer = os.Stdout
 
+func fib(i int) int {
+	var a = 0
+	var b = 1
+	fmt.Fprintln(out, b)
+	var sum int
+	for cnt := 0; cnt < i-1; cnt++ {
+		sum = a + b
+		fmt.Fprintln(out, sum)
+		a = b
+		b = sum
```

Comment:
You can update in one go: `a, b = b, sum`.

Reply:
Done. Thanks.

Reply:
It also occurred to me that you don't need sum at all. I'll let you figure it out. 🙂